javalib-ex-test-suite/src/test/scala/scala/scalajs/testsuite/javalibex/DataInputStreamTest.scala

```diff
@@ -2,329 +2,352 @@ package scala.scalajs.testsuite.javalibex
-        val stream = newStream(
-            0xbf, 0x80, 0x00, 0x00,
-            0x45, 0x8e, 0x9c, 0x83,
-            0x80, 0x00, 0x00, 0x00,
-            0x00, 0x00, 0x00, 0x00,
-            0x7f, 0xc0, 0x00, 0x00,
-            0x7f, 0x80, 0x00, 0x00,
-            0xbb, 0x03, 0x12, 0x6f,
-            0xff, 0x80, 0x00, 0x00,
-            0xff // dangling
-        )
-
-        expect(stream.readFloat()).toBeCloseTo(-1.0f, 6)
-        expect(stream.readFloat()).toBeCloseTo(4563.564f, 6)
-        expect(stream.readFloat()).toBeCloseTo(-0.0f, 6)
-        expect(stream.readFloat()).toBeCloseTo(0.0f, 6)
-        expect(stream.readFloat().isNaN).toBe(true)
-        expect(stream.readFloat()).toBe(Float.PositiveInfinity)
-        expect(stream.readFloat()).toBeCloseTo(-0.002f, 6)
-        expect(stream.readFloat()).toBe(Float.NegativeInfinity)
-        expect(() => stream.readDouble()).toThrow
-      }
+  @Test def should_provide_readFloat(): Unit = {
+    val stream = newStream(
+        0xbf, 0x80, 0x00, 0x00,
+        0x45, 0x8e, 0x9c, 0x83,
+        0x80, 0x00, 0x00, 0x00,
+        0x00, 0x00, 0x00, 0x00,
+        0x7f, 0xc0, 0x00, 0x00,
+        0x7f, 0x80, 0x00, 0x00,
+        0xbb, 0x03, 0x12, 0x6f,
+        0xff, 0x80, 0x00, 0x00,
+        0xff // dangling
+    )
+
+    assertEquals(-1.0f, stream.readFloat(), 1e-6f)
+    assertEquals(4563.564f, stream.readFloat(), 1e-6f)
+    assertEquals(-0.0f, stream.readFloat(), 1e-6f)
+    assertEquals(0.0f, stream.readFloat(), 1e-6f)
+    assertTrue(stream.readFloat().isNaN)
+    assertEquals(Float.PositiveInfinity, stream.readFloat(), 0f)
+    assertEquals(-0.002f, stream.readFloat(), 1e-6f)
+    assertEquals(Float.NegativeInfinity, stream.readFloat(), 0f)
+    assertThrows(classOf[EOFException], stream.readFloat())
+  }
 
-      it("should provide `readInt`") {
-        val stream = newStream(
-            0x00, 0x00, 0x00, 0x00,
-            0x7f, 0xff, 0xff, 0xff,
-            0xff, 0xff, 0xff, 0xfc,
-            0x00, 0x00, 0x00, 0x53,
-            0x00, 0x00, 0x89, 0xa2,
-            0xff, 0xfe, 0x82, 0xfd,
-            0x80, 0x00, 0x00, 0x00,
-            0x00, 0x00, 0x00, 0x01)
-
-        expect(stream.readInt()).toBe(0)
-        expect(stream.readInt()).toBe(Int.MaxValue)
-        expect(stream.readInt()).toBe(-4)
-        expect(stream.readInt()).toBe(83)
-        expect(stream.readInt()).toBe(35234)
-        expect(stream.readInt()).toBe(-97539)
-        expect(stream.readInt()).toBe(Int.MinValue)
-        expect(stream.readInt()).toBe(1)
-        expect(() => stream.readDouble()).toThrow
-      }
+  @Test def should_provide_readInt(): Unit = {
+    val stream = newStream(
+        0x00, 0x00, 0x00, 0x00,
+        0x7f, 0xff, 0xff, 0xff,
+        0xff, 0xff, 0xff, 0xfc,
+        0x00, 0x00, 0x00, 0x53,
+        0x00, 0x00, 0x89, 0xa2,
+        0xff, 0xfe, 0x82, 0xfd,
+        0x80, 0x00, 0x00, 0x00,
+        0x00, 0x00, 0x00, 0x01)
+
+    assertEquals(0, stream.readInt(), 0)
```

Comment:
What is the third argument when comparing `Int`s?
(same below with `Long`, `Short`, etc.)